### PR TITLE
fix: update axe-core to 4.2.1

### PR DESCRIPTION
**Pull request by michael-siek**

### CODE REVIEW

File: packages/webdriverjs/package.json [hunk 1]
```diff
@@ -76,10 +76,10 @@
     "typescript": "^4.1.5"
   },
   "dependencies": {
-    "axe-core": "^4.2.0"
+    "axe-core": "^4.2.1"
   },
   "peerDependencies": {
-    "selenium-webdriver": ">= 2.53.1"
+    "selenium-webdriver": "<= 4.0.0-beta.3 || >= 2.53.1"
```
Comment (stephenmathieson) [inline packages/webdriverjs/package.json:82]: Was this change intentional? Seems unrelated.

Comment (michael-siek) [reply packages/webdriverjs/package.json:82]: This was intentional to remove the warning message since we use a `prerelease` version of selenium-webdriver.

**Review by stephenmathieson — APPROVED** [review]

File: packages/webdriverjs/package.json (hunk 1 above)
Comment (dbjorge) [reply packages/webdriverjs/package.json:82]: This might be a little wider than you want; I think the `<= 4.0.0-beta.3` part would accept v1.0.0.

Comment (dbjorge) [reply packages/webdriverjs/package.json:82]: (you can test this locally using the same library npm uses for semver matching with `npx semver -r "<= 4.0.0-beta.3 || >= 2.53.1" "1.0.0"` - it outputs `1.0.0`, which indicates that it believes `1.0.0` matches the given range)

File: packages/cli/package.json [hunk 2]
```diff
@@ -41,7 +41,7 @@
   ],
   "dependencies": {
     "@axe-core/webdriverjs": "^4.2.0",
-    "axe-core": "^4.2.0",
+    "axe-core": "^4.2.1",
```
Comment (stephenmathieson) [inline packages/cli/package.json:44]: Just _another_ reminder: users were able to use 4.2.1 before this patch landed. Since we have the `^` here, we'll allowing npm to install a _range_ of versions, not exclusively 4.2.1 (previously 4.2.0).

**Review by stephenmathieson — APPROVED** [review]
